### PR TITLE
Add Julien Jerphanion to the team

### PR DESCRIPTION
**Pull request by jjerphan**
This also adds a small SVG image for Mastodon.

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Thanks!

File: julien.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            <div class="reseau">
+                <a target="_blank" href="https://twitter.com/jjerphan"><img class="twi-gith" src="img/twitter.svg"></a>
+                <a target="_blank" href="https://fosstodon.org/@jjerphan"><img class="twi-gith" src="img/mastodon.svg"></a>
+                <a target="_blank" href="https://github.com/jjerphan"><img class="twi-gith" src="img/github.svg"></a>
+            </div>
+            <p>@jjerphan</p>
+          </div>
+          <div class="description">
+            <p align="justify">
+              Julien is a Scientific Software Engineer at Quantstack.
```
Comment (JohanMabille) [inline julien.html:56]: Typo: QuantStack

File: julien.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+              He obtained a MSc. Computer Science &amp; Engineering from Université de Technologie de Compiègne and a
```
Comment (JohanMabille) [inline julien.html:57]: Nitpicking: He holds a MSc. [...]

**Review by JohanMabille — CHANGES_REQUESTED** [review]

**Comment by jjerphan** [comment]
I've addressed @JohanMabille's comments in https://github.com/QuantStack/quantstack.github.io/pull/103.

**Pull request merged**